src/main/java/com/fvv/bookstore/util/PropertiesUtil.java

```diff
@@ -0,0 +1,41 @@
+package com.fvv.bookstore.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Load a properties file.
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 2018</p>
+ * @version 1.0 	
+ *
+ */
+public class PropertiesUtil {
+	
+	private static Properties prop;
+	
+	static {
+		prop = new Properties();
+		
+		try (InputStream input = new FileInputStream("config.properties")) {
```

Comment:
put "config.properties" in a constant in Constant class

Reply:
ok